packages/api-client/src/components/Sidebar/SidebarButton.vue

```diff
@@ -2,11 +2,16 @@
 import ScalarHotkey from '@/components/ScalarHotkey.vue'
 import { ScalarButton } from '@scalar/components'
 
-const props = defineProps<{
-  click: () => void
-  hotkey?: string
-  isApp?: boolean
-}>()
+const props = withDefaults(
+  defineProps<{
+    click: () => void
+    hotkey?: string
+    isDesktop?: boolean
```

Comment:
Is there a reason to pass this still as a prop vs injecting it?

Reply:
we want to have the hotkey displayed on request sidebar (while hiding it on the others). prob should be done differently